geoengine_partner/views/geo_partner_view.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.ui.view" id="geo_partner_view_form">
+        <field name="name">geo_partner_form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <notebook>
+                <page string="Geoengine Map">
+                    <field name="geo_point" widget="geo_edit_map"
+                               groups="base_geoengine.group_geoengine_user,base_geoengine.group_geoengine_admin"/>
+                </page>
+            </notebook>
+        </field>
+    </record>
+
+    <record id="ir_ui_view_geo_partner" model="ir.ui.view">
+        <field name="name">Partner view</field>
+        <field name="arch" type="xml">
+            <geoengine version="7.0">
+                <field name="name" select="1"/>
+                <field name="title" />
+                <field name="function"/>
+                <field name="website"/>
+                <field name="zip"/>
+                <field name="city"/>
+                <field name="country_id"/>
+            </geoengine>
+        </field>
+        <field eval="16" name="priority"/>
+        <field name="model">res.partner</field>
+    </record>
+
+    <record id="action_partner_form_view3" model="ir.actions.act_window.view">
+        <field eval="20" name="sequence"/>
+        <field name="view_mode">geoengine</field>
+        <field name="view_id" ref="ir_ui_view_geo_partner"/>
+        <field name="act_window_id" ref="base.action_partner_form"/>
+    </record>
+
+    <record id="geoengine_vector_layer_res_partner" model="geoengine.vector.layer">
+        <field name="geo_field_id"
+            ref="geoengine_partner.field_res_partner_geo_point"/>
+        <field name="name">Partner location</field>
+        <field eval="6" name="sequence"/>
+        <field name="view_id" ref="ir_ui_view_geo_partner"/>
+        <field name="geo_repr">basic</field>
+        <field eval="1" name="nb_class"/>
+        <field name="begin_color">#FF680A</field>
+        <field name="symbol_binary">iVBORw0KGgoAAAANSUhEUgAAACAAAAAfCAYAAACGVs+MAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJ
```

Comment:
FYI is is possible to provide binary content as a relative path to the actual file in the repo.